fluency.js

```diff
@@ -1 +1,59 @@
+async function getUsers() {
+    const response = await fetch(`https://jsonplaceholder.typicode.com/users`)
+    const json = await response.json()
+    return json
+}
+console.log(getUsers())
 
+async function companyInfo() {
+    const data = await getUsers();
+    return data.map(users => {
```

Comment:
When setting high order array function parameters, it's best to call it as a singular item rather than a plural item. Instead of `users`, it would be better to call it `user`, because the functions loop through one item in an array at a time. Naming is important 😁